### ISSUE TITLE
Refine variable details page

### ISSUE THREAD

**Issue by rreitmann** [dzhw/metadatamanagement#766]
- rework order of statistical information (will be define by @kpstephan)
- add deviance and mode
- use "formulas" for statistical labels (see https://www.mathjax.org/)
- add maximize/minimize and copyToClipboard directly to labels of generationRule and filterRule
- rework related information (use links as defined by @kpstephan)
- harmonize with other details pages (see #790)

**Comment by kpstephan** [comment]
display the links in the following order
- Studie
- Frage
- Datensatz
- Erhebungen
- Publikationen

**Comment by rreitmann** [comment]
@kpstephan Please check the variable details page

**Comment by kpstephan** [comment]
results:

Detail Card, Eingangsfilter
- in small client windows (e.g. mobile browser) the buttons (clippboard, maximise/minimise) overlap the content

**Comment by kpstephan** [comment]
@limouri changes ok
@anfranken please fix the two last points above (at a later time)

**Comment by anfranken** [comment]
@kpstephan 
- modus is changed to mode
- I thought "Durchschnittliche Abweichung" should be an measure of dispersion for "ordinal" variables?
